bin/check-jenkins-job-status.rb

```diff
@@ -110,9 +110,10 @@ def job_status(job_name)
     status = jenkins_api_client.job.get_current_build_status(job_name)
     # If the job is currently running, get the status of the last build instead
     if status == 'running'
-      last_build = jenkins_api_client.job.get_current_build_number(job_name) - 1
-      build = jenkins_api_client.job.get_build_details(job_name, last_build)
-      status = build['result'].downcase
+      build = jenkins_api_client.job.get_build_details(job_name, 'lastCompletedBuild')
+      if build['result']
```

Comment:
based on what you said as far as using `lastCompletedBuild` is this needed? If it does not have a result what happens?

Reply:
In the case of 'lastCompletedBuild', the build['result'] should be not null.
It was more of a 'just in case' to avoid an exception, in which case it would return nothing.

Actually that makes me think that there might not be a `lastCompletedBuild` in the case of a new job, in which case `build` would be null ...